settings.py

```diff
@@ -0,0 +1,20 @@
+import os
+
+from pydantic.v1 import BaseSettings
+
+
+class Settings(BaseSettings):
+    PROJECT_NAME: str = "Temperatures in city"
+
+    DATABASE_URL: str | None = "sqlite+aiosqlite:///./city_temperature.db"
```

Comment:
The type hint `str | None` requires Python 3.10 or later. If you're using an earlier version, consider using `Optional[str]` from the `typing` module.